src/components/pages/taxonomy/list/table/taxon-modal/observation/use-observation-count.tsx

```diff
@@ -0,0 +1,35 @@
+import { axGetListData } from "@services/observation.service";
+import { useEffect } from "react";
+import { useImmer } from "use-immer";
+
+export default function useObsCount(taxonId) {
+  const [count, setCount] = useImmer({
```

Comment:
Since you are not mutating any complex data here using 2 seprate `useState` will keep this thing much simpler